lib/controller/page/type/page.confirmation/page.confirmation.controller.js

```diff
@@ -1,16 +1 @@
-const ConfirmationController = {}
-
-ConfirmationController.setContents = (pageInstance, userData, res) => {
```

Comment:
doesn't this log the user out by killing their session? if so where is this happening now since this has been removed?

Reply:
It's now in `route-metadata`. I simplified the code around how these controllers are called and created some distinct async functions to handle the calls into them (ensuring that any `Promises` are properly resolved, too, which wasn't happening. Which seems to be an AR thing).

There is now no need to pass the Express request `req` and response `res` objects into any of the controllers (since it was only _this_ controller which needed either of them) and the routing mechanism now decides that the session is done.

It does mean that `routes-metadata` has a specific test for the `page.confirmation` controller -- but we currently need specific tests in there for some of the other pages and components, anyway, so I figured it made more sense to standardise the "calling controllers" mechanism and keep the smell in `routes-metadata` for now. (If routing lives on a layer _above_ the page/component controllers, Express objects shouldn't need to go as deep as any of those controllers at all -- ideally, Express objects shouldn't need to be exposed even to the _routing_ layer -- so this is a way of starting to move that behaviour up a step.) 

Long-term all of the page/component-specific tests in `route-metadata` can die and the session can be cleared on _any_ exit page. For now, this is mostly to standardise and simplify the calls into the controllers.